src/Uno.UI.Wasm/ts/WindowManager.ts

```diff
@@ -690,7 +745,7 @@
 		): boolean {
 			const params = WindowManagerRegisterEventOnViewParams.unmarshal(pParams);
 
-			this.registerEventOnViewInternal(params.HtmlId, params.EventName, params.OnCapturePhase, params.EventFilterName, params.EventExtractorName);
+			this.registerEventOnViewInternal(params.HtmlId, params.EventName, params.OnCapturePhase, params.EventFilterId, params.EventExtractorId);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5247895)